src/api/db/job-run.ts

```diff
@@ -0,0 +1,50 @@
+import { JobRun } from './model';
+
+export function getRun(runId: number): Promise<any> {
+  return new Promise((resolve, reject) => {
+    new JobRun({ id: runId }).fetch()
+      .then(job => {
+        if (!job) {
+          reject();
+        }
+
+        resolve(job.toJSON);
+      });
+  });
+}
+
+export function insertJobRun(data: any): Promise<any> {
+  return new Promise((resolve, reject) => {
+    new JobRun(data).save(null, { method: 'insert' }).then(job => {
+      if (!job) {
+        reject();
+      }
+
+      resolve(job.toJSON());
+    });
+  });
+}
+
+export function updateJobRun(data: any): Promise<any> {
+  return new Promise((resolve, reject) => {
+    const runId = data.id;
+    new JobRun({ id: runId }).save(data, { method: 'update', require: false }).then(() => {
+      getRun(runId).then(job => resolve(job));
```

Comment:
You're right, it should be like that right
 `new JobRun({ id: data.id }).save(data, { method: 'update', require: false })
      .then(job => resolve(job));`